wikipedia-dark.user.css

```diff
@@ -545,6 +654,12 @@
   #footer-icons img {
     filter: brightness(70%);
   }
+  #mw-indicator-mw-helplink a {
+    filter: invert(80%) hue-rotate(180deg);
```

Comment:
Just apply straight colors to the selectors. Same with the rule below.

Reply:
Those are both images (set with `background-image`) so inverting them was the only way. In the particular case of the help link, there was no way to separate the text from the icon (usually, it's in the `::before` pseudoclass), meaning I had to fix the text color as well.

Reply:
This is what I did to it https://gist.github.com/the-j0k3r/8f8823e6c5ac654875a47fe8107e12d8 then I link it in my own style, you feel free to import that image into a base64 or into the repo and use that instead.

Reply:
Oh neat, I'll minimize and inline the SVG then (leaving a minor risk of eventual obsolescence).